src/Mechanisms/HandleRequests/HandleRequests.php

```diff
@@ -104,8 +105,21 @@ function handleUpdate()
             'assets' => SupportScriptsAndAssets::getAssets(),
         ];
 
-        $finish = trigger('response', $responsePayload);
```

Comment:
I think this `trigger()` is still needed, I suspect that is why the other tests are failing.

Reply:
Absolutely, I forgot a line and it was kind of working just because `$finish` was already defined earlier.
That fixes the other tests.